docs/examples/error_hander/README.rst

```diff
@@ -0,0 +1,153 @@
+Global Error Handler
+====================
+
+Overview
+--------
+
+This example shows how to use the global error handler.
+
+
+Preparation
+-----------
+
+This example will be executed in a separate virtual environment:
+
+.. code:: sh
+
+    $ mkdir global_error_handler
+    $ virtualenv global_error_handler
+    $ source global_error_handler/bin/activate
+
+Installation
+------------
+
+Here we install first ``opentelemetry-sdk``, the only dependency. Afterwards, 2
+error handlers are installed: ``error_handler_0``  will handle
+``ZeroDivisionError`` exceptions, ``error_handler_1`` will handle
+``IndexError`` and ``KeyError`` exceptions.
+
+.. code:: sh
+
+    $ pip install opentelemetry-sdk
+    $ git clone https://github.com/open-telemetry/opentelemetry-python.git
+    $ pip install -e opentelemetry-python/docs/examples/error_handler/error_handler_0
+    $ pip install -e opentelemetry-python/docs/examples/error_handler/error_handler_1
+
+Execution
+---------
+
+An example is provided in the
+``opentelemetry-python/docs/examples/error_handler/example.py``.
+
+You can just run it, you should get output similar to this one:
+
+.. code::
```

Comment:
There is actually a traceback lexer we can use here and throughout.

```suggestion
.. code:: pytb
```

Reply:
Nice! :+1: